contracts/Super721IMX.sol

```diff
@@ -58,6 +58,9 @@ contract Super721IMX is PermitControl, ERC165Storage, IERC721 {
   /// The public identifier for the right to disable item creation.
   bytes32 public constant LOCK_CREATION = keccak256("LOCK_CREATION");
 
+  /// The public identifier for the right to execute the mintFor fucntion.
```

Comment:
nit: rename this since it's not really gating the right to call `mintFor`, it's controlling the `toggleMintFor`.